src/app/store/reducers/formular-gui-reducer.ts

```diff
@@ -6,20 +6,10 @@ import { FormularEditorState, INITIAL_STATE } from '../states/formular-editor-st
 import { FormularGuiActions } from '../actions/formular-gui-actions';
 import { Form } from '../../data/forms/form';
 
-const updateBindings = (state: FormularEditorState, form: any[]): FormularEditorState => {
-    return tassign(state);
-};
-
-const updateCCText = (state: FormularEditorState): FormularEditorState => {
-    return tassign(state);
-};
-
 const updateValues = (state: FormularEditorState, form: Form): FormularEditorState => {
     return tassign(state, { form: form });
```

Comment:
Das Form-Objekt muss geklont werden, bevor es in den State geschrieben werden darf.